Civi/Api4/Generic/Result.php

```diff
@@ -190,15 +190,14 @@ public function setCountMatched(int $c) {
    * @param $name
    * @return array
    */
-  public function column($name) {
+  public function column($name): array {
     return array_column($this->getArrayCopy(), $name, $this->indexedBy);
   }
 
   /**
    * @return array
    */
-  #[\ReturnTypeWillChange]
-  public function jsonSerialize() {
+  public function jsonSerialize(): array {
```

Comment:
@seamuslee001 added the `#[\ReturnTypeWillChange]` annotation here for PHP8 compatibility, but according to the docs overriding a function with a more specific type is supported.
The parent interface is `public function jsonSerialize(): mixed` so overriding with `:array` is compatible.